src/components/social-share/social-share.scss

```diff
@@ -0,0 +1,7 @@
+.__social-share-container {
```

Comment:
Any reason we prefix this with ` .__`? Maybe something like this?
```suggestion
.socials-share-container {
```

Reply:
Also, in the design we have a `1px border-bottom`, can you please add that one as well?

Reply:
To avoid style collision.

Reply:
Makes sense. Can you please add the `border` tho? 😄